woocommerce/woocommerce-android
# Fixed crash when trying to edit negative stock quantity

Fixes #2073 by adding logic to support negative stock quantity value when editing products.

**This PR is in draft since it includes changes made in the `release/3.8` branch and should be merged into `develop` for the 3.9 beta release only when the 3.8 branch has been merged into `develop`.** cc @jkmassel 

#### Screenshots
<img width="300" src="https://user-images.githubusercontent.com/22608780/77304134-06453e80-6d1a-11ea-9679-2cbb4c00297c.gif"/>


#### To test
- Open `wp-admin` for a store and edit the stock quantity of a product to -5.
- Open the app and click on the `Products` tab.
- Click on the product you edited in step 1.
- Click on the `Inventory` section.
- Notice the stock quantity displayed is -5.
- Click on the stock quantity and remove the 5 from the text field.
- Notice the app no longer crashes. 
- Enter another amount such as 2 and click on `Done`.
- Verify that the stock quantity displayed is product detail is correct.
- It would also be good to test updating an empty stock quantity.

Update release notes:
